src/content/protocols/sky.md

```diff
@@ -0,0 +1,387 @@
+---
+protocol: "Sky"
+website: "https://sky.money/"
+x: "https://x.com/SkyEcosystem"
+github: ["https://github.com/makerdao"]
+defillama_slug: ["sky-lending"]
+chain: "Ethereum"
+stage: 0
+reasons: []
+risks: ["L", "H", "M", "H", "L"]
+author: ["mmilien_"]
+submission_date: "2025-03-11"
+publish_date: "1970-01-01"
+update_date: "1970-01-01"
+---
+
+# Summary
+
+The Sky protocol is a decentralised protocol developed around the `USDS` stablecoin. It is managed by the Sky ecosystem governance. The Sky Protocol features Sky tokens (`USDS`, `SKY`, `DAI`, `MKR`), the Sky Savings Rate, Sky Token Rewards, Activation Token Rewards, and SkyLink. The Protocol was built from the core module of the MakerDAO Protocol and replaces Maker. As Sky is built on top of Maker and both protocol's tokens are tied by fixed conversion rate, this review also includes the MakerDAO protocol and its contracts.
+Maker is a lending protocol centered around `DAI`. Users can borrow `DAI` using a list of accepted collaterals. Fixed rates allow users to go to and from legacy tokens at any time (`DAI` to `USDS`, `MKR` to `SKY`).
+
+# Overview
+
+## Chain
+
+The Sky protocol is currently mainly on Ethereum mainnet. As part of Sky's _End Game_ plan the protocol is gradually deploying on other chains and aims to launch its own chain in the future. This review focuses on the Ethereum mainnet deployment of Sky.
+
+> Chain score: Low
+
+## Upgradeability
+
+Core contracts such as `USDS` and `sUSDS` are upgradeable through governance actions. Governance has a complete control over the protocol and could arbitrarily mint tokens, liquidate positions, stop contracts, and steal collateral.
+Governance is controlled by `MKR`/`SKY` token holders who sealed their tokens to earn voting power which allows them to submit and vote on respective proposals, or delegate their votes to another user.
+
+> Upgradeability score: High
+
+## Autonomy
+
+The Sky protocol relies on the provider Chronicle for price feeds of collateral assets. An Oracle Security Module (`OSM`) enforces a 1-hour delay on price updates and the governance can freeze the current price to prevent further updates in case of emergency. The Chronicol protocol uses non-upgradeable contracts and a system of decentralized validators (and challengers) to push price updates on-chain. The price feed provider could be changed through a governance proposal.
```

Comment:
so according to our framework, if a dependency itself is "Stage 2" decentralized, then the defi protocol too earns a Low centralization score. So the question here is whether chronicle is "Stage 2" decentralized